apps/mobile/src/screens/PostScreen/usePost.tsx

```diff
@@ -80,21 +103,34 @@ export function usePost({ queryRef }: { queryRef: usePostFragment$key }) {
             'WorldwideFeedFragment_globalFeed'
           );
 
+          const communityStore = store.get(communityConnection);
+
           const connectionName = `${connectionId}(includePosts:${
             isPostEnabled ? 'true' : 'false'
```

Comment:
I see this is from a previous PR, but wonder if we can just assume `isPostEnabled` is true, if `usePost()` is being called